docker-bake.hcl

```diff
@@ -214,6 +214,17 @@ target "proofs-tools" {
   tags = [for tag in split(",", IMAGE_TAGS) : "${REGISTRY}/${REPOSITORY}/proofs-tools:${tag}"]
 }
 
+target "holocene-deployer" {
+  dockerfile = "./packages/contracts-bedrock/scripts/upgrades/holocene/upgrade.dockerfile"
+  context = "./packages/contracts-bedrock/scripts/upgrades/holocene"
+  args = {
+    REV = "cl/holocene-upgrade-package"
```

Comment:
We will need to follow up here to update the revision the docker image includes, once we've cut the new tag for the `op-contracts` release. This release will need to contain the `op-program` absolute prestate for holocene in the deploy configs as well.

Reply:
Just to be super clear, the release will not include _THE_ absolute prestate for holocene, it needs to include _A_ prestate for holocene for the chains that are using the config from the release.  Chains that upgrade to holocene later will need a _different_ prestate with the activation time for their chain baked in.

There's been a lot of confusion around this so want to be super clear that the required prestate may vary from chain to chain (but that it is also sometimes ok for it to be the same across chains).

Reply:
What I mean is, for _our_ deploy configs (the ones housed in the `contracts-bedrock/deploy-configs` folder), we'll want to set this revision to a tag that has the prestate updated to a Holocene-compatible version of `op-program`. Otherwise whoever deploys the Holocene upgrade for OP Mainnet/OP Sepolia/OP Sepolia devnet will need to do that themselves.

Other chain operators will use their own deploy config passed through the `DEPLOY_CONFIG_PATH` envvar where they can update the prestate hash to a version of the program that supports their chain.

Reply:
hmm, I thought the contracts releases had already been tagged as `op-contracts/v1.8.0-rc.1`. The absolute prestate wouldn't have been updated for all chains anyway so we will need an easy way to specify it.